src/components/home-comps/border-beam/index.module.less

```
@@ -6,7 +6,11 @@
   width: 100%;
   height: 100%;
   overflow: hidden;
-  border-radius: 24px;
+  // Match the host card's rounded corners so the rectangular canvas
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)